packages/core/src/shared/systemUtilities.ts

```diff
@@ -49,6 +49,17 @@ export class SystemUtilities {
     private static gitPath: string
     private static bashPath: string
 
+    public static getOperatingSystem(): 'MAC' | 'WINDOWS' | 'LINUX' {
```

Comment:
A quick search shows that this exact code is in `core/src/codewhisperer/client/codewhisperer.ts`. Can you update that class as well to use this util

Reply:
Yes, I know. I specially omitted that as it is not my vertical but still thought of having this in the shared folder.